searxng_extra/update/update_tracker_patterns.py

```diff
@@ -0,0 +1,36 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+"""Fetch trackers"""
+
+import json
+import httpx
+
+from searx.data import data_dir
+
+DATA_FILE = data_dir / "tracker_patterns.json"
+CLEAR_LIST_URL = "https://raw.githubusercontent.com/ClearURLs/Rules/refs/heads/master/data.min.json"
+
+
+def fetch_clear_url_filters():
+    resp = httpx.get(CLEAR_LIST_URL)
+    if resp.status_code != 200:
+        # pylint: disable=broad-exception-raised
+        raise Exception(f"Error fetching ClearURL filter lists, HTTP code {resp.status_code}")
+
+    providers = resp.json()["providers"]
+    rules = []
+    for rule in providers.values():
+        rules.append(
+            {
+                "urlPattern": rule["urlPattern"].replace("\\\\", "\\"),  # fix javascript regex syntax
+                "exceptions": [exc.replace("\\\\", "\\") for exc in rule["exceptions"]],
+                "trackerParams": rule["rules"],
+            }
+        )
```

Comment:
When PR https://github.com/searxng/searxng/pull/4836 has been merged we have a boilerplate to manage `data` in a *ExpireCache*.

.. and instead of the json file which needs to be created today, we can directly read the rules from `CLEAR_LIST_URL` in the *ExpireCahe* .. 

ping @dalf: Here's an example where SearXNG uses `data` from external projects. To ensure efficient access to the `data` objects, these objects are loaded into an (expiring) cache.  This cache is not yet implemented in this PR, but its related to what we discuss in #4836.